base/show.jl

```diff
@@ -1614,6 +1614,15 @@ function show(io::IO, src::CodeInfo; debuginfo::Symbol=:source)
     print(io, ")")
 end
 
+# Show for inference limit trace objects
+function show(io::IO, ecl::Core.Compiler.EdgeCycleLimited)
```

Comment:
We don't typically like `Base` to have any references to Core.Compiler. Perhaps this should go into the IRShow module?